app/frontend/src/components/form/form.test.js

```diff
@@ -0,0 +1,37 @@
+import { disableInputs, enableInputs } from './form';
```

Comment:
nice, well done 👍  few minor comments below

Reply:
you could add a test for the click event and that it calls the correct method enable/disable

Reply:
remember we used spys to spy on another function so you can see if/how many times/what arguments the function you are spying on is called with